### PR TITLE
Meeting 59.md

### PR DESCRIPTION
**Pull request by poojaranjan**
Notes #59

### CODE REVIEW

File: All Core Devs Meetings/Meeting 59.md [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+
+**DECISION 58.3**: Reseach will be updated in the comments in the agenda.
+
+**Status**: Done
+
+**ACTION 58.1** Cat Herders to look at updating EIP1.
+
+**Tim**: I know there's been some stuff happening I've personally spent a lot of time there trying to describe the process and sort of gather all the various conversations that have happened around this. I don't know if anyone has anything to add. 
+
+**Status**: WIP
```
Comment (timbeiko) [inline All Core Devs Meetings/Meeting 59.md]: Not sure if we want to add it, but the associated GH issue is https://github.com/ethereum-cat-herders/PM/issues/19

File: All Core Devs Meetings/Meeting 59.md [hunk 2]
```diff
@@ -0,0 +1,451 @@
...
+# Date for next meeting
+
+April 26, 2019 at 14:00 UTC
+
+# Attendees
+
+* Alex B.
+* Borris Mann (Fellowship of Ethereum Magicians)
+* Brooklyn Zelenka (SPADE)
+* Danno Ferrin
```
Comment (timbeiko) [inline All Core Devs Meetings/Meeting 59.md]: Danno is part of PegaSys

File: All Core Devs Meetings/Meeting 59.md [hunk 3]
```diff
@@ -0,0 +1,451 @@
...
+* Joesph Delong (Pegasys)
+* John Adler
+* Martin Holst (Ethereum)
+* Matthew Halpern
+* Mikhail Kalinin (EthereumJ/Harmony)
+* Pawel Bylica
+* Peter (Geth)
+* Phil
+* Pooja Ranjan (Ethereum Cat Herders)
+* Tim Beiko (Pantheon)
```
Comment (timbeiko) [inline All Core Devs Meetings/Meeting 59.md]: I am part of PegaSys (Pantheon is our product).

File: All Core Devs Meetings/Meeting 59.md [hunk 4]
```diff
@@ -0,0 +1,451 @@
...
+Already covered.
+
+# 7. Client Updates (only if they are posted in the comments below)
+
+## 7a) Geth
+No update.
+
+## 7b) Parity Ethereum
+**phillux**: [update](https://github.com/ethereum/pm/issues/93)
+Just wanted to add Parity Ethereum recently added quick consensus. SO full support for Gorli testnet is in. That is in 2.5 beta.
```
Comment (veox) [inline All Core Devs Meetings/Meeting 59.md]: > quick consensus

Clique consensus. ;)

File: All Core Devs Meetings/Meeting 59.md [hunk 5]
```diff
@@ -0,0 +1,451 @@
...
+
+## 3b) Please add more EIPs to the agenda
+
+**Tim**: Talking about EIPs there were two posted in the comments of this call. But I know there's also a whole lot that's just mentioned. So perhaps we have Bryant and Pooja who proposed the comments on the call. So maybe it makes sense for them to talk about it. Bryant that he actually was first in the comments so if you want to go ahead and talk about [EIP 1344](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1344.md).
+
+**Bryant**: Sure thing. I am not the author of this one but about like a week and a half ago I was playing around with some 712 stuff and I was like why isn't there a chain ID opcode and so I went to go look and there was a proposal for one. So I've been helping kind of format in a nicer way and get it accepted and kind of gather feedback. Its pretty simple for the most. There have been a little bit of a discussion around like what kind of data types do we represent. I want to gather more feedback, that what else we need to add this to Istanbul?
```
Comment (fubuloubu) [inline All Core Devs Meetings/Meeting 59.md]:
```suggestion
**Bryant**: Sure thing. I am not the author of this one but about like a week and a half ago I was playing around with some 712 stuff and I was like why isn't there a chain ID opcode and so I went to go look and there was a proposal for one. So I've been helping kind of format in a nicer way and get it accepted and kind of gather feedback. Its pretty simple for the most. There have been a little bit of a discussion around like what kind of data types do we represent it with. I want to gather more feedback, what else do we need to add this to Istanbul?
```

Comment (fubuloubu) [reply All Core Devs Meetings/Meeting 59.md]: grammar

File: All Core Devs Meetings/Meeting 59.md [hunk 6]
```diff
@@ -0,0 +1,451 @@
...
+
+**Pooja**: Yeah, I mean we can write it up. But I'm open for suggestions over here. Like if everyone agrees that Ethmagician is the only forum where it should be discussed, it can be changed accordingly.
+
+**Bryant**:  I think it makes sense that people have said that EthMagician is going to catch all environment for discussion. Some people may view it too technical. So..
+
+**Boris**: Hang on Bryant, let's just figure out who cares about this discussion, have the discussion and come back with a proposal in the form of an EIP. 
+
+**Bryant**: Sure, But I guess the last thing I wanted to say is does it make sense to say the 'discussions to' is the only place for EIP conversation be captured?
+
+**Boris**: NO
```
Comment (fubuloubu) [inline All Core Devs Meetings/Meeting 59.md]: Reminder to myself to follow up on this with @bmann

**Comment by mariapaulafn** [comment]
Hey @poojaranjan dont have edit access, so [i just prepared the summary here](https://docs.google.com/document/d/1www4jmuHPHEYWAJhOjLJE9ye5B6MHh215leB-9H3e6E/edit?usp=sharing)

**Pull request merged**